plugins/livelog/livelog.go

```diff
@@ -79,7 +79,7 @@ func (tp *taskPlugin) Prepare(context *runtime.TaskContext) error {
 
 	err := tp.context.CreateRedirectArtifact(runtime.RedirectArtifact{
 		Name:     "public/logs/live.log",
-		Mimetype: "text/plain",
+		Mimetype: "text/plain; charset=utf-8",
```

Comment:
Probably best to make this a constant...
`var mimePlainText = "text/plain; charset=utf-8"`

And then reuse that everywhere... Just so that we only have it once this package...